config/install/core.entity_view_display.paragraph.localgov_link.default.yml

```diff
@@ -2,28 +2,39 @@ langcode: en
 status: true
 dependencies:
   config:
+    - field.field.paragraph.localgov_link.localgov_link_style
     - field.field.paragraph.localgov_link.localgov_title
     - field.field.paragraph.localgov_link.localgov_url
     - paragraphs.paragraphs_type.localgov_link
+  module:
+    - options
 id: paragraph.localgov_link.default
 targetEntityType: paragraph
 bundle: localgov_link
 mode: default
 content:
+  localgov_link_style:
+    type: list_key
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    weight: 2
+    region: content
   localgov_title:
-    weight: 0
+    type: string
     label: above
     settings:
       link_to_entity: false
     third_party_settings: {  }
-    type: string
+    weight: 0
     region: content
   localgov_url:
-    weight: 1
+    type: string
     label: above
```

Comment:
I think we should remove the label from above here, it's jarring on the backend when editing the CMS.

Reply:
This one has been removed as well.